portal/static/js/src/admin.js

```diff
@@ -498,7 +520,10 @@ import CurrentUser from "./mixins/CurrentUser.js";
                 if (this.isAdminUser()) {
                     return false;
                 }
-                this.handleMedidataRave(); //a function specifically created to handle MedidataRave related stuff
+                //a function specifically created to handle MedidataRave related stuff
```

Comment:
Can you move these comments (definitely appreciated) into the body of each function, maybe add a bit of detail about the business need?